src/Sulu/Bundle/ContentBundle/Document/RouteDocument.php

```diff
@@ -52,6 +52,16 @@ class RouteDocument implements
     private $targetDocument;
 
     /**
+     * @var \DateTime
+     */
+    private $created;
+
+    public function __construct()
+    {
+        $this->created = new \DateTime();
```

Comment:
The same issue is in @wachterjohannes PR. I suggest making an attribute on the Metadata to say if the document is localized or not, the behavior would then depend on that. But that is only an idea.